### PR TITLE
Merge `next` into `master`

### PR DESCRIPTION
**Pull request by jsumners**
As titled.

### CODE REVIEW

**Review by gurgunday — APPROVED** [review]
lgtm

File: package.json [hunk 1]
```diff
@@ -29,21 +29,21 @@
   },
   "homepage": "https://github.com/fastify/fastify-csrf#readme",
   "dependencies": {
-    "@fastify/csrf": "^6.0.0",
-    "@fastify/error": "^3.0.0",
-    "fastify-plugin": "^4.0.0"
+    "@fastify/csrf": "^6.3.0",
```
Comment (jsumners) [inline package.json:32]:
```suggestion
    "@fastify/csrf": "^7.0.0-pre.fv5.1",
```

File: package.json [hunk 2]
```diff
@@ -29,21 +29,21 @@
...
+    "@fastify/error": "^3.4.1",
+    "fastify-plugin": "^4.5.1"
```
Comment (jsumners) [inline package.json:34]: Need to publish these first.

**Review by jsumners — COMMENTED** [review]
Blocking.

File: package.json [hunk 3]
```diff
@@ -29,21 +29,21 @@
...
-    "@fastify/secure-session": "^7.0.0",
-    "@fastify/session": "^10.3.0",
-    "@types/node": "^20.1.0",
-    "fastify": "^4.2.0",
+    "@fastify/cookie": "^9.3.1",
+    "@fastify/pre-commit": "^2.1.0",
+    "@fastify/secure-session": "^7.5.0",
+    "@fastify/session": "^10.7.0",
+    "@types/node": "^20.12.7",
+    "fastify": "^4.26.2",
```
Comment (jsumners) [inline package.json:42]: Blocking again: need to update all of these to pre releases.

**Pull request merged**